datalad/distributed/create_sibling_gitlab.py

```diff
@@ -190,14 +191,17 @@ class CreateSiblingGitlab(Interface):
             site)""",
             constraints=EnsureStr() | EnsureNone()),
         publish_depends=publish_depends,
+        dry_run=Parameter(
+            args=("--dry-run",),
+            action="store_true",
+            doc="""if set, no repository will be created, only tests for
+            name collisions will be performed, and would-be repository names
+            are reported for all relevant datasets"""),
         dryrun=Parameter(
             args=("--dryrun",),
             action="store_true",
-            doc="""If this flag is set, no communication with GitLab is
-            performed, and no repositories will be created. Instead
-            would-be repository names and configurations are reported for all
-            relevant datasets
-            """),
+            doc="""Deprecated. Use the renamed
+        [CMD: --dry-run CMD][PY: `dry_run` PY] parameter""")
```

Comment:
Is this a whitespace issue?

Reply:
Thanks for catching this - a left over from copy-pasting. Should be resolved now.